.github/workflows/build.yml

```diff
@@ -10,18 +10,26 @@ jobs:
   build:
     name: Build
     runs-on: ubuntu-latest
+    timeout-minutes: 60
     steps:
       - uses: actions/checkout@v4
         with:
           fetch-depth: 100
+      - run: jq '.packageManager' package.json | tr -d '"pnpm@'
+        id: package-manager-version
```

Comment:
I didn't want to have to keep the pnpm version in sync between the definition in `package.json` so I just pulled the value I wanted from the file, and parse out the version number.

So, version number for PNPM is defined by the `packageManager` field in the `package.json` and `.nvmrc` defines the node version. So, if you have nvm with corepack enabled, it should just always use pnpm instead of npm